index.js

```diff
@@ -181,7 +180,11 @@ function expand(str, isTop) {
       N.push(c);
     }
   } else {
-    N = concatMap(n, function(el) { return expand(el, false) });
+    N = [];
+
+    n.map(function(el) {
+      [].push.apply(N, expand(el, false));
```

Comment:
probably also faster

```suggestion
      N.push.apply(N, expand(el, false));
```